numpy/core/_add_newdocs.py

```diff
@@ -1084,12 +1084,25 @@
 
     Examples
     --------
-    >>> x = np.arange(6).reshape(2,3)
-    >>> np.ascontiguousarray(x, dtype=np.float32)
-    array([[0., 1., 2.],
-           [3., 4., 5.]], dtype=float32)
+    Try to change memory layout of ndarray from C-contiguous to C-contiguous::
+    >>> x = np.ones((2, 3))          # Create C-contiguous ndarray
+    >>> y = np.ascontiguousarray(x)  # Convert `x` to C-contiguous ndarray
```

Comment:
I would prefer to avoid those trailing comments, i.e. rather write:
```
Starting with a Fortran contiguous array:

>>> x = np.ones((2, 3), order='F')
>>> x.flags['F_CONTIGUOUS']
True

Calling ``ascontiguousarray`` makes a C-contiguous copy:

>>> y = np.ascongiuosuarray(x)
>>> y.flags['C_CONTIGUOUS']
True
>>> np.may_share_memory(x, y)
False
```

Reply:
Done.